frontend/packages/dsco/src/fontAwesomeV6Icon/stories/FontAwesomeV6Icon.story.tsx

```diff
@@ -0,0 +1,169 @@
+import {Meta, StoryFn} from '@storybook/react';
+
+import FontAwesomeV6Icon, {FontAwesomeV6IconProps} from '../index';
+
+export default {
+  title: 'DesignSystem/FontAwesomeV6Icon',
```

Comment:
Also not a thing to fix right now, but not sure if we need the "Design System" category in Storybook any more.

Reply:
Correct. I keep it same way is to was before just to save some time, plus we'll probably need to replace it with atom/molecule/organism soon, so since it'll be updated anyways decided to keep as is for now